quests.json

```
@@ -4083,7 +4366,51 @@
                 "rep": 0.03
             }
         ],
-        "objectives": [],
+        "objectives": [
+            {
+                "type": "build",
+                "target": "59d6088586f774275f37482f",
+                "number": 1,
+                "with": [
+                    {
+                        "type": "stat",
+                        "name": "durability",
+                        "value": ">=60"
+                    },
+                    {
+                        "type": "stat",
+                        "name": "ergonomics",
+                        "value": ">=45"
+                    },
+                    {
+                        "type": "stat",
+                        "name": "recoil",
+                        "value": "<=350"
+                    },
+                    {
+                        "type": "stat",
+                        "name": "sighting range",
+                        "value": ">=800"
+                    },
+                    {
+                        "type": "attachment",
+                        "name": "Suppressor"
+                    },
+                    {
+                        "type": "attachment",
+                        "name": "Fortis Shift tactical grip",
+                        "id": "59f8a37386f7747af3328f06"
+                    },
+                    {
+                        "type": "attachment",
+                        "name": "AK 7.62x39 Magpul PMAG 30 GEN M3 30-round magazine",
+                        "id": "59d6272486f77466146386ff"
```

```suggestion
                        "type": "part",
                        "id": "59f8a37386f7747af3328f06"
                    },
                    {
                        "type": "part",
                        "id": "59d6272486f77466146386ff"
```

Same thoughts here about differentiating attachment categories and specific parts